### PR TITLE
abricotine 0.7.0

### PR DESCRIPTION
**Pull request by dmlemos**
After making all changes to the cask:

- [X] `brew cask audit --download {{cask_file}}` is error-free.
- [X] `brew cask style --fix {{cask_file}}` reports no offenses.
- [X] The commit message includes the cask’s name and version.
- [X] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Comment by vitorgalvao** [comment]
> It seems the App source '/usr/local/Caskroom/abricotine/0.7.0/Abricotine.app' is not there.

[How to fix](https://github.com/Homebrew/homebrew-cask/blob/master/doc/reporting_bugs/source_is_not_there_error.md).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]
> It seems the App source '/usr/local/Caskroom/abricotine/0.7.0/Abricotine.app' is not there.

[How to fix](https://github.com/Homebrew/homebrew-cask/blob/master/doc/reporting_bugs/source_is_not_there_error.md).

**Comment by dmlemos** [comment]
@vitorgalvao Thanks for the tip. Fixed and also added zap contents.

File: Casks/abricotine.rb [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
+  sha256 'ccd48b7967dabb6e59872010222f60f455ac195b3c549c206d0c1ed3237fc3c2'
 
   # github.com/brrd/Abricotine was verified as official when first introduced to the cask
   url "https://github.com/brrd/Abricotine/releases/download/#{version}/Abricotine-#{version}-osx-x64.zip"
   appcast 'https://github.com/brrd/Abricotine/releases.atom'
   name 'abricotine'
   homepage 'https://abricotine.brrd.fr/'
 
-  app 'Abricotine.app'
+  app 'Abricotine-darwin-x64/Abricotine.app'
```
Comment (vitorgalvao) [inline Casks/abricotine.rb]: This in enough; we won’t copy the directory.

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**